recipes/libavro/meta.yaml

```diff
@@ -0,0 +1,37 @@
+package:
+    name: libavro
+    version: 1.7.7
+
+source:
+    fn: avro-c-1.7.7.tar.gz
+    url: http://www-us.apache.org/dist/avro/avro-1.7.7/c/avro-c-1.7.7.tar.gz
+    sha1: cbb698682d662c5e0abec023dcd37ce1f3db80d4
+    patches:
+        - make.patch
```

Comment:
Please explain the patch in a comment above it. Has this patch been submitted upstream. If not, it should be.